airbyte-ci/connectors/pipelines/pipelines/helpers/github.py

```diff
@@ -152,7 +152,7 @@ def create_or_update_github_pull_request(
                 content = base64.b64encode(file.read()).decode("utf-8")  # Encode file content to base64
                 blob = repo.create_git_blob(content, "base64")
                 changed_file = ChangedFile(path=str(modified_file), sha=blob.sha)
-        changed_files.append(changed_file)
+            changed_files.append(changed_file)
```

Comment:
Given where it's assigned, you might have to indent it more, so it's all the way within the `with` block?